h4-labeling-and-versioning/convert-datasets.ipynb

```diff
@@ -0,0 +1,317 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pathlib import Path\n",
+    "from pprint import pprint\n",
+    "\n",
+    "from tqdm import tqdm"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Convert WebGazer dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
```

Comment:
why do we need this? could you convert this to the script, please?

Reply:
That's a little ad-hoc notebook to convert raw data from two open-source datasets into the suitable format. I'll convert it to py script.

Reply:
Converted into two separate scripts at `./scripts`.